tests/test_connections.py

```diff
@@ -0,0 +1,9 @@
+import urllib.request
+import pytest
+
+SITE_LOCATION = "http://localhost:5000"
+
+def test_connection():
+    assert urllib.request.urlopen(SITE_LOCATION).getcode() == 200
+
+
```

Comment:
blank lines at the end of the file are usually flagged as errors by tools like pyflakes or pep8, other ten that it now looks like the code would work. Did you run it?